code/modules/jobs/job_types/garrison/town_elder.dm

```diff
@@ -0,0 +1,529 @@
+	H.equip_to_slot(thiefcloak, SLOT_CLOAK, TRUE)
+
+
+/datum/advclass/town_elder/battlemage
+	name = "Mage's Guild Battlemage "
+
+	tutorial = "You always dreamed of studying magic, but your family couldn’t afford the Academy. Instead, you took up the sword, spending years as a mercenary. But fate had other plans. Seeing your potential, the crown offered you a place in the Mage's Guild, on the condition that you serve the realm. Now, you stand as both scholar and soldier, wielding magic not for glory, but for the kingdom."
+	outfit = /datum/outfit/job/town_elder/battlemage
+
+	//Mercenary turned into a mage, works in the Mages Guild for the crown.
+
+	category_tags = list(CTAG_TOWN_ELDER)
+
+
+/datum/outfit/job/town_elder/battlemage/pre_equip(mob/living/carbon/human/H)
+	pants = /obj/item/clothing/pants/trou/leather
+	armor = /obj/item/clothing/shirt/robe/newmage/sorcerer
+	shirt = /obj/item/clothing/armor/chainmail/hauberk
+	shoes = /obj/item/clothing/shoes/boots
+	gloves = /obj/item/clothing/gloves/leather
+	neck = /obj/item/clothing/neck/gorget
+	backl = /obj/item/storage/backpack/satchel
+	backr = /obj/item/weapon/sword/long
+	belt = /obj/item/storage/belt/leather
+	beltr = /obj/item/storage/magebag/apprentice
+	beltl = /obj/item/flashlight/flare/torch/lantern
+	backpack_contents = list(/obj/item/storage/belt/pouch/coins/mid = 1, /obj/item/storage/keyring/battlemage = 1, /obj/item/book/granter/spellbook/apprentice = 1)
+
+
+	if(H.mind)
+		H.mana_pool.set_intrinsic_recharge(MANA_ALL_LEYLINES)
+		H.mind?.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
+		H.mind?.adjust_skillrank(/datum/skill/magic/arcane, 2, TRUE)
+		H.mind?.adjust_skillrank(/datum/skill/craft/cooking, 1, TRUE)
+		H.mind?.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
+		H.mind?.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
+		H.mind?.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+		H.mind?.adjust_skillrank(/datum/skill/misc/athletics, 4, TRUE)
+		H.mind?.adjust_skillrank(/datum/skill/combat/swords, 3, TRUE)
+
+		H.change_stat(STATKEY_STR, 1)
+		H.change_stat(STATKEY_INT, 2)
+		H.change_stat(STATKEY_END, 1)
+		H.change_stat(STATKEY_SPD, 1)
+
+		if(H.age == AGE_OLD)
+			H.mind?.adjust_skillrank(/datum/skill/magic/arcane, 1, TRUE)
+			H.mind?.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
+			H.change_stat(STATKEY_INT, 1)
+
+		H.mind.adjust_spellpoints(5)
+		H.mind.AddSpell(new /obj/effect/proc_holder/spell/targeted/touch/prestidigitation)
+
+
+
+	ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
+
+
+
+/datum/advclass/town_elder/crowns_acolyte
+	name = "Crown's Acolyte"
+
+	tutorial = "As an Acolyte, you dedicated your life to others, never yourself. When you saved a noble, they repaid you with a home and gold, but you saw it as the will of the Ten. Stepping away from the Church, you caught the crown’s eye. Now, you serve not the faith, but the realm, living for the first time on your own terms."
```

Comment:
since they stepped away from the institution of the Church but still have a faith
```suggestion
	tutorial = "As an Acolyte, you dedicated your life to others, never yourself. When you saved a noble, they repaid you with a home and gold, but you saw it as the will of the Ten. Stepping away from the Church, you caught the crown’s eye. Now, you serve not the Church, but the realm, living for the first time on your own terms."
```

Reply:
true